CHANGES.md

```diff
@@ -1,6 +1,18 @@
 Release Changes List
 ====================
 
+
+version 1.22.1
+==============
+
+Update to twenty-fifth public release of Chapel, June 2020
+(see also changes below for 1.22.0)
+
+Portability
+-----------
+* addressed some coding issues that caused problems with gcc 10.x
```

Comment:
This reads odd to me. Maybe "Fixed portability issues with GCC 10"?

Reply:
To me, that makes it sound like gcc 10 itself has portability issues.  How about

```
* fixed portability issues in our code w.r.t. gcc 10.x
```

Reply:
works for me